docs/xxshownativestacksymbols.md

```diff
@@ -0,0 +1,44 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:[+|-]ShowNativeStackSymbols
+
+This option controls whether Java&reg; dumps show the names of functions in native call stacks.
+
+## Syntax
+
+        -XX:-ShowNativeStackSymbols
+        -XX:+ShowNativeStackSymbols=<value>
+
+| Setting                    | Value  |Effect|Default|
+|----------------------------|--------|------|:-------:|
+| `-XX:-ShowNativeStackSymbols` |       | Don't show native stack symbols, not even in response to a signal, such as a fault signal      |   |
+| `-XX:+ShowNativeStackSymbols` | basic | Show all the available native stack symbols for a signaled or faulting thread, but only easily acquired native stack symbols for other threads     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>|
+| `-XX:+ShowNativeStackSymbols` | all | Show all the available native stack symbols for all threads     |   |
```

Comment:
Please insert the spaces requested, for example
```
| Setting                    | Value  |Effect|Default|
```
should be
```
| Setting                    | Value  | Effect | Default |
```
and on the "basic" line, there should be a space before the final `|`.

Reply:
Done. Please check. Thanks!